### PR TITLE
LZWCodec documentation complete

### PR DESCRIPTION
**Pull request by IronMan988**
SOLVES #1 

All documentation written.
Please review.
@MigDinny

### CODE REVIEW

File: lzw.py [hunk 1]
```diff
@@ -27,17 +26,39 @@
...
         pass
-    
+
+
     def encode_LZW(self,data):
+        """
+        Encodes a given data using LZW.
+
+        Parameters:
+            data (one-dimensional array): the data to be encoded
```
Comment (MigDinny) [inline lzw.py:47]: (array) is preferred over (one-dimensional array)

File: lzw.py [hunk 2]
```diff
@@ -27,17 +26,39 @@
...
+
+        Returns:
+            compressed_data: LZW encoded data
```
Comment (MigDinny) [inline lzw.py:50]: Comments on returns must be in this format:

Returns:
     string: LZW encoded data

ONLY THE DATATYPE!

File: lzw.py [hunk 3]
```diff
@@ -48,107 +69,155 @@ def encode_LZW(self,data):
...
-        maximum_table_size = pow(2,int(size))
+        """
+        Decodes a given data using LZW.
+
+        Parameters:
+            data (one-dimensional array): the data to be decoded
+            size (int): 2^size is the max size of the dictionary
+
+        Returns:
+            compressed_data: LZW decoded data
```
Comment (MigDinny) [inline lzw.py:139]: same here

**Review by MigDinny — CHANGES_REQUESTED** [review]
Check the comments and commit again so ill review again

**Comment by IronMan988** [comment]
Review again

**Review by MigDinny — APPROVED** [review]

**Pull request merged**